models/_generate_layers.py

```diff
@@ -81,7 +81,8 @@ def setup():
         if param.is_module_init_arg():
           print(param.get_module_param_docstring(indent="    "), file=f)
       print('    """', file=f)
-      print(f"    super().__init__({'**kwargs' if issubclass(cls_base, LayerBase) else ''})", file=f)
+      params = get_super_call_params(layer_class)
```

Comment:
`params` is a bad variable name (ambiguous or misleading - you would expect a dict, not a str or sth else). Use `params_str` or better `kwargs_repr` or so.